e2e-playwright/README.md

```diff
@@ -0,0 +1,66 @@
+## Quick Start
+
+To start the development, run:
+
+```
+npm install
+npm run install:browsers (only for the first time)
+npm run test (or npm run test:chrome for browser specific)
+```
+
+where `new-project` is your desired folder. This should result in cloning the project, setting up your repository as
```

Comment:
hm, I am not sure if I didn't misunderstood but I see that you're referencing `new-project` but it is missing in commands 🤔

Reply:
Yeah i copy-paste part of root README.md and i forgot to remove it. TY.